eras/mary/impl/src/Cardano/Ledger/Mary/PParams.hs

```diff
@@ -6,6 +6,7 @@ module Cardano.Ledger.Mary.PParams () where
 import Cardano.Ledger.Core
 import Cardano.Ledger.Crypto
 import Cardano.Ledger.Mary.Era (MaryEra)
+import Cardano.Ledger.Shelley.Governance
```

Comment:
did you mean this open import?

Reply:
I didn't care much about it, but I'll make it explicit